### DISCUSSION TITLE
Habit tracker Lua script

### DISCUSSION THREAD

**Discussion by MathieuRhayne** [mpurses/Sonder#48] [Q&A] [answered]
So I followed the comments in the +5Habits.ini deleting the comment modifier ';'  on habits 6-10 and deleting the last two comment modifiers at the end of the script. step 3. edit calendar lua script. I opened in notepad, and followed comment on habit 5 that says replace 'if' with 'elseif' if you add habits 6-10. saved, refreshed and no extra habits, so edited again and deleted the comment modifier '--' on all of the habits hence 6-10. saved, refresh, nothing. any help?

**Comment by mpurses** [comment, marked as answer]
Assuming you changed all in the code correctly (it looks like it) - Next open up Variables.inc in "Documents\Rainmeter\Skins\Sonder\@Resources" folder and change lines 239 and below to what you need. And change **Habit6Disabled** (through 10) to 0.

**Reply by MathieuRhayne** [reply]
Thank you, it worked! this is game changer!!!!!!
